cmd/importer/db.go

```diff
@@ -277,7 +277,7 @@ func genColumnData(table *table, column *column) (string, error) {
 
 		data = append(data, '\'')
 		return string(data), nil
-	case mysql.TypeNewDecimal, mysql.TypeDecimal:
```

Comment:
We should remove the TypeDecimal here, it's misuse

Reply:
revert done, thx~

ditto.